src/drivers/imu/bmi055/BMI055_gyro.cpp

```diff
@@ -334,30 +333,22 @@ BMI055_gyro::ioctl(struct file *filp, int cmd, unsigned long arg)
 					bool want_start = (_call_interval == 0);
 
 					/* convert hz to hrt interval via microseconds */
-					unsigned ticks = 1000000 / arg;
+					unsigned interval = 1000000 / arg;
 
 					/* check against maximum rate */
-					if (ticks < 1000) {
+					if (interval < 1000) {
```

Comment:
The BMI055 max sample rate is 2000, (https://www.mouser.com/ds/2/783/BST-BMI055-DS000-08-786482.pdf page 16), maybe we add a #define here as well to get rid of another magic number... I know these calls are going to be deprecated soon, so if it's not worth it I understand.

Reply:
Followup issue?